test/parallel/test-fs-options-immutable.js

```diff
@@ -58,9 +58,7 @@ if (common.canCreateSymLink()) {
   );
 }
 
-if (!common.isAix) {
-  // TODO(thefourtheye) Remove this guard once
-  // https://github.com/nodejs/node/issues/5085 is fixed
+{
```

Comment:
All the other tests also have this block thing. I thought it would be consistent. If you want to remove it, I can do it.